src/components/Movie/MovieDetails.js

```diff
@@ -0,0 +1,62 @@
+import React, {Component} from 'react';
+
+import Header from '../Header/Header';
+import Footer from '../Footer/Footer';
+import SimilarMovies from './SimilarMovies';
+import MovieBanner from './MovieBanner';
+import './moviedetails.css';
+
+export default class MovieDetails extends Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            showSearchButtonOnHeader: true,
+            movie: {},
+            selectedGenre: '',
+            similarMovies: []
+        };
+    }
+    componentDidMount() {
+        let url = 'http://react-cdp-api.herokuapp.com/movies/'+this.props.match.params.id;
+        fetch(url)
+            .then(res => res.json())
+            .then(response => {
+                this.setState({
+                    movie: response
+                });
+                this.fetchSimilarMovies(response.genres[0]);
+            });
+    }
+    componentWillReceiveProps(nextProps) {
+        let url = 'http://react-cdp-api.herokuapp.com/movies/'+nextProps.match.params.id;

```

Comment:
You can use an file with your constants with the base API url to be easier in case of a refactoring

Reply:
URL moved to constants file.